src/integrationTest/java/integration/tests/SystemEngineTest.java

```diff
@@ -33,10 +35,15 @@
 public class SystemEngineTest extends IntegrationTest {
 
 	private static final long ID = ProcessHandle.current().pid() + System.currentTimeMillis();
-	private static final String DATABASE_NAME = "jdbc_system_engine_integration_test_" + ID;
-	private static final String ENGINE_NAME = DATABASE_NAME + "_engine";
+	private static final String SECOND_DATABASE_NAME = "jdbc_system_engine_integration_test_" + ID;
+	private static final String ENGINE_NAME = SECOND_DATABASE_NAME + "_engine";
 	private static final String ENGINE_NEW_NAME = ENGINE_NAME + "_2";
 
+	private static final String USE_DATABASE_NAME = "jdbc_use_db_" + ID;
```

Comment:
Now this is outside of scope, but why do we generate the ID here? We already have a unique database name provided to the test suite in `-Ddb=`. We can just do that db name + "system_engine". At least this way we'll be able to match system engine test dbs to a particular JDBC test run and keep test db prefixes consistent.

Reply:
The ID is needed here because If test failed or interrupted for some reason the DB is still there and the next execution of the test fails. I saw it several times when running test locally. Also clash can happen when running intetegration tests twice. This happened to me too: I ran tests, fixed someting and ran them again. One of the executions failed because other one created DB first.

Reply:
Then those tests are setup incorrectly, we should look into that. `integration-setup-action` on each run provides a unique db & engine name. Otherwise it's a hack that's there because some tests are not using the db name provided.